CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ contribution. See the [DCO](DCO) file for details.
 # Email and Chat
 
 The project currently uses the general etcd email list and IRC channel:
-- Email: [coreos-dev](https://groups.google.com/forum/#!forum/etcd-dev)
+- Email: [coreos-dev](https://groups.google.com/g/etcd-dev)
 - IRC: #[coreos](irc://irc.freenode.org:6667/#etcd) IRC channel on freenode.org
```

Comment:
s/coreos/etcd? :)

Reply:
Should it be like that?

>- Email: [etcd-dev](https://groups.google.com/g/etcd-dev)
>- IRC: #[etcd](irc://irc.freenode.org:6667/#etcd) IRC channel on freenode.org

I think, we have the same error in dbtester too.

Reply:
Is IRC still being used?

Reply:
> Is IRC still being used?

No idea :D. I personally don't use it.

Reply:
should be fixed now

Reply:
Pls let me know if you want to remove or correct IRC channel too :)